langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -21,9 +21,7 @@ configurable boolean managerEnabled = false;
 # Config to specify transaction log directory
 configurable string logBase = "transaction_log_dir";
 
-# Information about a transaction that does not change
-# after the transaction is started.
-public type Info readonly & record {|
+public type InfoInternal record {|
```

Comment:
may be we have to make this not public

Reply:
We cannot make this non-public as it gives me `"Attemp to refer to non-public.. "` error